ha_mroonga.cpp

```diff
@@ -14953,7 +14953,105 @@ mrn_io_and_cpu_cost ha_mroonga::scan_time()
   DBUG_RETURN(time);
 }
 
-#ifdef MRN_ENABLE_WRAPPER_MODE
+#ifdef MRN_HANDLER_HAVE_KEYREAD_TIME
+#  ifdef MRN_ENABLE_WRAPPER_MODE
```

Comment:
Can we avoid overriding entirely without wrapper mode like the following?

```cpp
#if defined(MRN_HANDLER_HAVE_KEYREAD_TIME) && defined(MRN_ENABLE_WRAPPER_MODE)
IO_AND_CPU_COST ha_mroonga::wrapper_rnd_pos_time(ha_rows rows) {}
IO_AND_CPU_COST ha_mroonga::storage_rnd_pos_time(ha_rows rows) {}
IO_AND_CPU_COST ha_mroonga::rnd_pos_time(ha_rows rows) {}
IO_AND_CPU_COST ha_mroonga::wrapper_keyread_time() {}
IO_AND_CPU_COST ha_mroonga::storage_keyread_time() {}
IO_AND_CPU_COST ha_mroonga::keyread_time() {}
#endif
```

Reply:
They are used when:
* wrapper mode is enabled on build time
* storage mode is used on run time

Reply:
Oh. I see. I understood them. Thanks.